integreat_cms/cms/templates/analytics/_page_hix_widget.html

```diff
@@ -0,0 +1,36 @@
+{% extends "_collapsible_box.html" %}
+{% load i18n %}
+{% load static %}
+{% load rules %}
+{% load content_filters %}
+{% block collapsible_box_icon %}
+    gauge
+{% endblock collapsible_box_icon %}
+{% block collapsible_box_title %}
+    {% translate "Page HIX list" %}
+{% endblock collapsible_box_title %}
+{% block collapsible_box_content %}
+    <div class="flex flex-wrap justify-between gap-2">
+        <p class="py-3">{% translate "The list below shows pages ascendingly ordered by HIX value." %}</p>
+        <p class="py-3">{{ ready_for_mt }} % {% translate " of all pages are ready for automatic translation." %}</p>
```

Comment:
```suggestion
        <p><b>{{ ready_for_mt }}</b> % {% translate " of all pages are ready for automatic translation." %}</p>
```
Also not here. Instead I would add some highlighting for the percentage, since this is an important information IMO

Reply:
```suggestion
        <p><b>{{ ready_for_mt }}%</b> {% translate " of all pages are ready for automatic translation." %}</p>
```
Good idea, but I'd include the percentage sing in the bold text.